config/locales/forms/registration_number_forms/en.yml

```diff
@@ -4,7 +4,7 @@ en:
       new:
         title: Companies House registration number
         heading:
-          limitedCompany: What's the registration number of the company?
+          limitedCompany: What’s the company registration number?
           limitedLiabilityPartnership: What's the registration number of the limited liability partnership?
```

Comment:
Should we be changing the format of the LLP text to match?

Reply:
I am really not sure what you are asking. What's LLP?

Reply:
Limited Liability Partnership. For example changing it to, "What's the limited liability partnership registration number?"

Reply:
I'll ask Rose :)

Reply:
I'll wait for Rose to update the ticket and open a different PR for that change. Merging this now :)